### PR TITLE
Feature/language filter

### PR DESCRIPTION
**Pull request by pierman1**
No svg's yet, but it's working.

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ function setupState(state, emitter) {
   Object.assign(state, {
     machineslider: {
       active: false,
-      items: ['genre', 'type', 'keyword', 'location', 'author', 'hardsort'],
+      items: ['genre', 'type', 'keyword', 'location', 'author', 'hardsort', 'language'],
```
Comment (Mimaaa) [inline app/index.js]: can you move language before hardsort and author before location?

**Review by Mimaaa — CHANGES_REQUESTED** [review]
pls fix

File: app/index.js (hunk 1 above)
Comment (Mimaaa) [reply app/index.js]: can you also add the svg's before we merge this into master?

Comment (pierman1) [reply app/index.js]: genre --> type --> keyword --> language --> author --> sort --> location

**Review by Mimaaa — APPROVED** [review]
🍔

**Pull request merged**